pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -632,17 +635,21 @@ func TestEnoughRequests(t *testing.T) {
 			draExtendedResourceEnabled: true,
 			pod: newResourcePod(
 				framework.Resource{MilliCPU: 1, Memory: 1, ScalarResources: map[v1.ResourceName]int64{extendedResourceDRA: 1}}),
-			nodeInfo:                  framework.NewNodeInfo(newResourcePod(framework.Resource{MilliCPU: 0, Memory: 0})),
-			name:                      "extended resource backed by DRA",
+			nodeInfo: framework.NewNodeInfo(newResourcePod(framework.Resource{MilliCPU: 0, Memory: 0})),
+			name:     "extended resource backed by DRA",
+			// When DRAExtendedResource is enabled and there's a matching DeviceClass,
+			// the noderesources plugin delegates to DRA instead of reporting insufficient resources.
 			wantInsufficientResources: []InsufficientResource{},
 		},
 		{
 			draExtendedResourceEnabled: true,
+			nilDRAManager:              true,
 			pod: newResourcePod(
 				framework.Resource{MilliCPU: 1, Memory: 1, ScalarResources: map[v1.ResourceName]int64{extendedResourceDRA: 1}}),
```

Comment:
here you don't want to use extendedResourceDRA, instead, use extendedResourceA. otherwise, the global cache will have the mapping, and it will be satisfied.